src/Infrastructure/Elastic/ElasticIndexAdapter.php

```diff
@@ -110,7 +109,7 @@ public function ensureIndex(IndexConfigurationInterface $indexConfiguration): vo
     {
         $exists = $this->client->indices()->exists([
             'index' => $indexConfiguration->getWriteIndexName(),
-        ]);
+        ])->asBool();
```

Comment:
👀 nice feature [asBool()](https://www.elastic.co/blog/introducing-the-new-php-client-for-elasticsearch-8)

> We also offer asObject(), asString(), and asBool() functions to serialize the body as an object of the standard class of PHP ([stdClass](https://www.php.net/manual/en/language.types.object.php)), as string or boolean (true if 2xx response, false otherwise).